src/java/net/java/sen/SenFactory.java

```diff
@@ -125,10 +125,10 @@ private static InputStream getInputStream(String name, String dictionaryDir) thr
 
   private final String[] posIndex, conjTypeIndex, conjFormIndex;
   private final ByteBuffer costs, pos, tokens, trie;
-  
-  
+
   public static final String unknownPOS = "未知語";
-  
+  public static boolean compatibilityMode = true;
```

Comment:
"compatibilityMode" is affect only Unknown Katakana token, right?
I think we should change "compatibilityMode" to "tokenizeUnknownKatakana" or something else.